sdk/lib/cmlib/hivebin.c

```diff
@@ -16,7 +16,7 @@ HvpAddBin(
 {
     PHMAP_ENTRY BlockList;
     PHBIN Bin;
-    SIZE_T BinSize;
+    ULONG BinSize;
```

Comment:
Ok, but then you need to remove some explicit (ULONG) casts that have been made when BinSize is used in this function.

Reply:
Fixed